sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Release.java

```diff
@@ -19,15 +19,16 @@
  */
 package org.sonar.updatecenter.common;
 
+import com.google.common.base.Function;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.google.common.base.Function'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)